packages/storage/src/__tests__/queryExtensions.test.ts

```diff
@@ -29,6 +29,8 @@ vi.mock('node:fs', () => ({
   mkdirSync: vi.fn(),
 }))
 
+vi.mock('fs/promises')
```

Comment:
`node:fs/promises` is already mocked further up the file. Isn't that enough?

Reply:
If I remove I get ``` ❯ src/__tests__/queryExtensions.test.ts  (17 tests | 1 failed) 52ms
         × Query extensions > create > create will save files
           → Cannot read properties of undefined (reading 'mockResolvedValue')
       ✓ src/adapters/FileSystemStorage/FileSystemStorage.test.ts  (4 tests) 11ms
       ✓ src/adapters/MemoryStorage/MemoryStorage.test.ts  (6 tests) 3ms
      
      ⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯
      
       FAIL  src/__tests__/queryExtensions.test.ts > Query extensions > create > create will save files
      TypeError: Cannot read properties of undefined (reading 'mockResolvedValue')```

Let me see if can refactor.

Reply:
I wasn't able to but had comments added for purpose.